renderers/abc_renderer.js

```diff
@@ -4,7 +4,13 @@ const abc = require('abcjs');
 function renderAbc(str, opts) {
   const div = document.createElement('div');
   abc.renderAbc(div, str, {
-    visualTranspose: opts.transpose
+    visualTranspose: opts.transpose,
+    staffwidth: opts.maxWidth,
```

Comment:
What happens when `opts.maxWidth === undefined`?

Reply:
Uses the default of 740.